cmake/targets.cmake

```diff
@@ -447,6 +447,7 @@ function(wn_filesystem_files name)
     add_custom_target(${name} DEPENDS 
        ${CMAKE_CURRENT_BINARY_DIR}/${PARSED_ARGS_OUTPUT_DIR}/${name}.h
        ${CMAKE_CURRENT_BINARY_DIR}/${PARSED_ARGS_OUTPUT_DIR}/${name}.cpp)
+    overlay_named_file(cmake/target_functions/post_filesystem_files.cmake)
```

Comment:
WHat does this add exactly?

Reply:
This lets us put the wn_filesystem_files into overlay directories.